README.md

```diff
@@ -1,17 +1,79 @@
 ## Kubeflow Tensorboards Operator
 
 ### Overview
-This charm encompasses the Kubernetes Python operator for Kubeflow Tensorboards (see
-[CharmHub](https://charmhub.io/?q=kubeflow-tensorboards)).
+This charm encompasses the Kubernetes Python operator for Kubeflow Tensorboards (see [CharmHub](https://charmhub.io/?q=kubeflow-tensorboards)).
 
 The Kubeflow Tensorboards operator is a Python script that wraps the latest released version
-of Kubeflow Tensorboards, providing lifecycle management and handling events such as install,
-upgrade, integrate, and remove.
+of Kubeflow Tensorboards, providing lifecycle management and handling events such as install, upgrade, integrate, and remove.
 
-## Install
+More information on Tensorboard can be found [on the upstream website](https://www.tensorflow.org/tensorboard).
 
-To install Kubeflow Tensorboards, run:
+## Usage
 
-    juju deploy kubeflow-tensorboards
+To get started, install Kubeflow lite bundle (See [Quick start guide](https://charmed-kubeflow.io/docs/quickstart)).
 
+Then, deploy Tensorboard Controller and Tensorboard Web App with
+```bash
+juju deploy tesnorboard-controller --trust
+juju deploy tensorboards-web-app --trust
```

Comment:
is `--trust` required for these charms?

Reply:
I also believe we should point to `--channel=latest/edge` here, this is the README for main so with latest/edge we actually deploy what currently on main.